hal/armv7m/imxrt/117x/Makefile

```diff
@@ -18,6 +18,7 @@ PLO_COMMANDS = alias app call console copy dump echo go help kernel kernelimg ma
 
 include devices/usbc-cdc/Makefile
 include devices/uart-imxrt117x/Makefile
-include devices/flash-imxrt/Makefile
+# include devices/flash-imxrt/Makefile
```

Comment:
Please remove this line if you don't use it anyway.

Reply:
Done.